### PR TITLE
Restructure project, move src files into src

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

**Comment by ekmartin** [comment]
I guess `public/js/main.js` shouldn't be in the repo? It was probably added since `*.js` has been removed from `.gitignore`.

**Comment by relekang** [comment]
You are right, it is removed now

File: Makefile [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
-	$(GULP)
+	$(PARSE)
+	$(STYLUS)
 	forever restart $(PWD)/index.js
 else
 	@echo "Not in a production environment!"
 endif
 
-.PHONY: test node_modules parse
+.PHONY: test install parse clean production all
```
Comment (ekmartin) [inline Makefile]: Is `all` a PHONY-target?

Comment (ekmartin) [inline Makefile]: Nevermind, I guess it is since `parse` is too :)

**Comment by ekmartin** [comment]
:tada:

**Pull request merged**